highlight/highlight
# Add error frequency graph to metrics tab

## Summary
The error monitoring metrics tab now includes a bar chart that displays the frequency of an error group. Currently, this distribution is fixed to 30 days, but a follow up will allow the user to select their time frame.

Changes to the information card:
- currently no way to exit out to help fill the while space
- Change "app" -> "environment"
- One button "Learn more" that leads to documentation
- Move below occurrences 

[Figma](https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?node-id=1%3A41&t=6SVuI6zkws4pddWi-0)

## How did you test this change?
Click test:
1. Load an error group (`/1/errors/:secure_id`)
2. Switch to the Metrics tab
- [ ] Confirm the graph is correct
- [ ] Confirm the total occurrences is correct
3) Hover over a column
- [ ] Confirm occurrence count is correct
4) Test on another error

![Screen Shot 2022-11-30 at 10 55 14 AM](https://user-images.githubusercontent.com/17744174/204848914-b54e4836-cb58-4f1d-a961-772689de629a.png)

## Are there any deployment considerations?
Tab is now enabled
